### ISSUE TITLE
use only point releases, disable beta version support

### ISSUE THREAD

**Issue by kls0e** [itzg/docker-minecraft-bedrock-server#243]
Since today, my server reports `1.19.10-beta20` which is too new for our iOS clients to connect to (currently 1.18.31). I know there is a `beta preview` for iOS but it is closed, and I would like the server to only run the latest point release. `-e VERSION=1.18` results in a 404, and -e `VERSION=PREVIOUS` results in 1.17.

**Comment by kornysietsma** [comment]
Just to note you can specify `VERSION=1.18.33.02` - you need a specific point release - I found the correct server version at https://minecraft.fandom.com/wiki/Bedrock_Edition_1.18.33

**Comment by itzg** [comment]
@kls0e please provide your container config, especially what you used for `VERSION`, and the logs. What you describe doesn't seem possible since it locates the version from https://www.minecraft.net/en-us/download/server/bedrock ...unless they accidentally published the beta in place of the release build.

**Comment by rippo** [comment]
Think they did, same issue for me
```
Looking up latest version...
Downloading Bedrock server version 1.19.10.20 ...
Backing up behavior_packs into backup-pre-1.19.10.20
Backing up definitions into backup-pre-1.19.10.20
Backing up minecraftpe into backup-pre-1.19.10.20
Backing up resource_packs into backup-pre-1.19.10.20
Backing up structures into backup-pre-1.19.10.20
Backing up treatments into backup-pre-1.19.10.20
Backing up world_templates into backup-pre-1.19.10.20
Starting Bedrock server...
```

**Comment by itzg** [comment]
Ah, I didn't notice the big red note just earlier 😀. Sounds like explicitly setting `VERSION` will be required for a bit.

**Comment by mirosm** [comment]
> Just to note you can specify `VERSION=1.18.33.02` - you need a specific point release - I found the correct server version at https://minecraft.fandom.com/wiki/Bedrock_Edition_1.18.33

Hi @kornysietsma, did you managed to make it run with -e VERSION=1.18.33.02? No matter what I use it keeps using the beta version. @itzg I checked the bedrock-entry.sh file and it seems to me that the case statement only goes to version 1.17. After that it defaults to the LATEST version, in this case the beta. Ah, thank you for everything, my family already had a lot of fun thanks to you! 😀

**Comment by itzg** [comment]
There's a wildcard/catch-all at the end that leaves VERSION as given

https://github.com/itzg/docker-minecraft-bedrock-server/blob/c82dc81222bbf320b54a85cf794f3624912592d3/bedrock-entry.sh#L81

**Comment by kornysietsma** [comment]
> Hi @kornysietsma, did you managed to make it run with -e VERSION=1.18.33.02? 
Yes - note I'm not using docker-compose, just a very simple commandline script:

```
docker run -d -it --name korny_minecraft \
  -e EULA=TRUE -e GAMEMODE=survival -e SERVER_NAME=my_server -e DIFFICULTY=easy  \
  -e VERSION=1.18.33.02 \
  -v /home/korny/minecraft/data:/data \
  -p 19132:19132/udp itzg/minecraft-bedrock-serve
```

**Comment by SeanPaulE** [comment]
I can confirm that adding the **_VERSION=1.18.33.02_** to the docker-compose downgrades the server and launches the expected version.

**Comment by mirosm** [comment]
> > Hi @kornysietsma, did you managed to make it run with -e VERSION=1.18.33.02?
> 
> Yes - note I'm not using docker-compose, just a very simple commandline script:
> 
> ```
> docker run -d -it --name korny_minecraft \
>   -e EULA=TRUE -e GAMEMODE=survival -e SERVER_NAME=my_server -e DIFFICULTY=easy  \
>   -e VERSION=1.18.33.02 \
>   -v /home/korny/minecraft/data:/data \
>   -p 19132:19132/udp itzg/minecraft-bedrock-serve
> ```
It worked for me too, thanks @kornysietsma!

**Comment by DominusExult** [comment]
in a docker-compose file you need to have the line
```
environment:
   VERSION: "1.18.33.02"
```
